.changeset/little-pianos-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@microsoft/atlas-css': major
```

Comment:
Because the external-facing files (.scss / .css) connected to Atlas have no changed in any meaningful way with this PR, I believe that no changeset is required. I had to look this up myself, since I wasn't sure, and found this discussion of the topic:

>If you completely hide/internalize all types in the dependency, then the version update to 1.2.4 or 1.3.0 should reflect your level of changes needed to consume the dependency.

https://github.com/semver/semver/issues/148

Reply:
Just noting to myself that with the exception of the above, this PR is looking good.

Reply:
Changeset removed